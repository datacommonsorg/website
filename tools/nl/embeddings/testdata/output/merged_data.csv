dcid,sentence
SV_1,xyz1;desc1;some text for SV1;abc1; SV1 palm text sentence;palm text for SV1; SV1 more text;name1
SV_2,desc2;some text for SV2;abc2; SV2 more text;xyz2;name2;even more text for SV2
SV_3,override3
SV_4,desc4;name4;xyz4;abc4
