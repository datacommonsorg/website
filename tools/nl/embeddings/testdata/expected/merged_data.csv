dcid,sentence
SV_1,SV1 more text;SV1 palm text sentence;abc1;desc1;name1;palm text for SV1;some text for SV1;xyz1
SV_2,SV2 more text;abc2;desc2;even more text for SV2;name2;some text for SV2;xyz2
SV_3,override3
SV_4,abc4;desc4;name4;xyz4
